frontend/src/components/roboDesignation/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import {ReactComponent as Robologo} from './spot.svg'
+import './style.css'
+
+const RoboDesignation = () => {
+  return (
+    <div className='robo-designation'><Robologo className='robo-designation__logo'/>Spot 1234 869 Full Name</div>
```

Comment:
Let's move a text into a store with a default value, as it'll be changed on a settings screen.

Reply:
the store will be created within another issue #17